llama-index-packs/llama-index-packs-raft-dataset/README.md

```diff
@@ -0,0 +1,47 @@
+# Corrective Retrieval Augmented Generation Llama Pack
+
+This LlamaPack implements the Corrective Retrieval Augmented Generation (CRAG) [paper](https://arxiv.org/pdf/2401.15884.pdf)
+
+Corrective Retrieval Augmented Generation (CRAG) is a method designed to enhance the robustness of language model generation by evaluating and augmenting the relevance of retrieved documents through a an evaluator and large-scale web searches, ensuring more accurate and reliable information is used in generation.
+
+This LlamaPack uses [Tavily AI](https://app.tavily.com/home) API for web-searches. So, we recommend you to get the api-key before proceeding further.
+
+### Installation
+
+```bash
+pip install llama-index llama-index-tools-tavily-research
```

Comment:
wrong pip install?

Reply:
Oh I think it got mixed up with CRAG. Will clean it up.